src/components/deductions/F1098eInfo.tsx

```diff
@@ -87,12 +88,15 @@ export default function F1098eInfo(): ReactElement {
   )
 
   return (
-    <form tabIndex={-1} onSubmit={onAdvance}>
-      <FormProvider {...methods}>
+    <FormProvider {...methods}>
+      <form tabIndex={-1} onSubmit={handleSubmit(onAdvance)}>
+        <Helmet>
+          <title>1098-E Information | Deductions | UsTaxes.org</title>
```

Comment:
Would it be better for all these titles to be in one place in the [Main component](https://github.com/ustaxes/UsTaxes/blob/master/src/components/Main.tsx#L122)